app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -28,6 +28,16 @@ def update_os_from_facts
             content_facet.kickstart_repository_id = valid_repos.first
           end
         end
+
+        def remote_execution_proxies(provider, *_rest)
```

Comment:
This method doesn't seem to be getting called (my breakpoint isn't hit when kicking off a REX job)

Reply:
Not sure what your environment looks like, but if it is a hijacked production setup, then all the dynflow-sidekiq@* services need to be cycled to pick up the changes.

Reply:
Just the usual katello devel environment - centos7-katello-devel.

I guess it's that these extensions are prepended first, and then the REX extensions are prepended and effectively overriding this method?

::Host.first.remote_execution_proxies(nil) isn't hitting this method in the console :(